pet-shop-web/src/test/java/throne/springreacto/petshopspring5/controller/PetControllerTest.java

```diff
@@ -64,7 +64,7 @@ void initCreationForm() throws Exception {
                 .andExpect(status().isOk())
                 .andExpect(model().attributeExists("owner"))
                 .andExpect(model().attributeExists("pet"))
-                .andExpect(view().name("pets/createOrUpdatePetForm"));
+                .andExpect(view().name("pet/createOrUpdatePetForm"));
```

Comment:
As convention --> Path name for resources such as pet will be named in singular form!